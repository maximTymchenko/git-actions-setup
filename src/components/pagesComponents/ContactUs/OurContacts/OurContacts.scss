@import '../../../../styles/vars';

.contacts {
    &__wrapper {
        @media screen and (max-width: $desktop) {
            margin-top: 50px;
        }
    }
    &__links {
        display: flex;
        flex-direction: column;
        margin-bottom: 30px;
    }

    &__container {
        display: flex;
        flex-direction: column;

        @media screen and (max-width: $desktop) {
            flex-direction: row;
            justify-content: space-between;
        }

        @media screen and (max-width: $tablet) {
            flex-direction: column;
        }
    }

    &__phone {
        display: block;
        text-decoration: none;
        font-family: 'OktaNeue-Normal';
        font-size: 16px;
        line-height: 1.69;
        color: $text__body;
        margin-bottom: 15px;
    }

    &__email {
        text-decoration: none;
        font-family: 'OktaNeue-Black';
        font-size: 15px;
        font-weight: 900;
        letter-spacing: 0.5px;
        color: $primary__light;
        margin-bottom: 10px;
    }
}
