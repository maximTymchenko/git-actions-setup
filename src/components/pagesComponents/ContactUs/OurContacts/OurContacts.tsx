import React, { FC } from 'react';
import PrivacyTitle from '../../../common/SectionTitle';
import './OurContacts.scss';

const Contacts: FC = () => {
    return (
        <div data-testid="our-contacts" className="contacts__wrapper">
            <div className="contacts__container">
                <div className="contacts__links">
                    <PrivacyTitle type="primary" title="New business" />
                    <a href="tel:+380931042757" className="contacts__phone">
                        +380–93–104–27–57
                    </a>
                    <a href="mailto:newbiz@blackbird-lab.com" className="contacts__email">
                        newbiz@blackbird-lab.com
                    </a>
                </div>
                <div className="contacts__links">
                    <PrivacyTitle type="primary" title={'Headquater'} />
                    <a href="tel:+380931042757" className="contacts__phone">
                        +380–93–104–27–57
                    </a>
                    <span className="contacts__phone">16 Shota Rustaveli St Kyiv,</span>
                    <span className="contacts__phone">02000 Ukraine</span>
                    <a href="mailto:info@blackbird-lab.com" className="contacts__email">
                        info@blackbird-lab.com
                    </a>
                </div>
                <div className="contacts__links">
                    <PrivacyTitle type="primary" title={'Social'} />
                    <a
                        target="_blank"
                        href="https://ua.linkedin.com/company/blackbird-lab"
                        className="contacts__email"
                    >
                        Linkedin
                    </a>
                    <a
                        target="_blank"
                        href="https://jobs.dou.ua/companies/blackbird-lab/"
                        className="contacts__email"
                    >
                        DOU
                    </a>
                </div>
            </div>
        </div>
    );
};

export default Contacts;
