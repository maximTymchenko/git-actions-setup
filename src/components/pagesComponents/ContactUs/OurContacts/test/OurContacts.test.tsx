import React from 'react';
import OurContacts from '../OurContacts';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

it('renders elem correctly', () => {
    const { getByTestId } = render(<OurContacts></OurContacts>);
    expect(getByTestId('our-contacts')).toBeTruthy();
});

it('renders without unexpected changes', () => {
    const tree = renderer.create(<OurContacts></OurContacts>).toJSON();
    expect(tree).toMatchSnapshot();
});
