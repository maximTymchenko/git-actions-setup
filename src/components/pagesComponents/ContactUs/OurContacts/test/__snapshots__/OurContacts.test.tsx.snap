// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without unexpected changes 1`] = `
<div
  className="contacts__wrapper"
  data-testid="our-contacts"
>
  <div
    className="contacts__container"
  >
    <div
      className="contacts__links"
    >
      <h2
        className="title title--primary"
        data-testid="section-title"
      >
        New business
      </h2>
      <a
        className="contacts__phone"
        href="tel:+380931042757"
      >
        +380–93–104–27–57
      </a>
      <a
        className="contacts__email"
        href="mailto:newbiz@blackbird-lab.com"
      >
        newbiz@blackbird-lab.com
      </a>
    </div>
    <div
      className="contacts__links"
    >
      <h2
        className="title title--primary"
        data-testid="section-title"
      >
        Headquater
      </h2>
      <a
        className="contacts__phone"
        href="tel:+380931042757"
      >
        +380–93–104–27–57
      </a>
      <span
        className="contacts__phone"
      >
        16 Shota Rustaveli St Kyiv,
      </span>
      <span
        className="contacts__phone"
      >
        02000 Ukraine
      </span>
      <a
        className="contacts__email"
        href="mailto:info@blackbird-lab.com"
      >
        info@blackbird-lab.com
      </a>
    </div>
    <div
      className="contacts__links"
    >
      <h2
        className="title title--primary"
        data-testid="section-title"
      >
        Social
      </h2>
      <a
        className="contacts__email"
        href="https://ua.linkedin.com/company/blackbird-lab"
        target="_blank"
      >
        Linkedin
      </a>
      <a
        className="contacts__email"
        href="https://jobs.dou.ua/companies/blackbird-lab/"
        target="_blank"
      >
        DOU
      </a>
    </div>
  </div>
</div>
`;
